Flick/SceneDelegate.swift

```diff
@@ -33,8 +33,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         var rootViewController: UIViewController
         if let token = AccessToken.current,
             !token.isExpired,
-            let _ = userDefaults.data(forKey: Constants.UserDefaults.user),
-            let _ = userDefaults.string(forKey: Constants.UserDefaults.authorizationToken)  {
+            let _ = userDefaults.string(forKey: Constants.UserDefaults.authorizationToken) {
```

Comment:
Did you remove user from user defaults? I think I kept it in case we wanted to save some basic user info in UserDefaults (idk if that may be useful...) so then we don't have to always rely on fetching info from backend?

Reply:
oh i didn't remove user from user defaults but we're just not checking it to see if we should show homeVC. 
The reason why re-running the simulator keep showing the FB "Log Out" before was bc the user user default was nil somehow. There might be a flaw in the way we stored user in user default so we also need to look into that when we refactor the whole FB login thing.